src/sentry/api/serializers/rest_framework/sentry_app.py

```diff
@@ -113,6 +118,27 @@ def validate_allowedOrigins(self, value):
                 raise ValidationError("'*' not allowed in origin")
         return value
 
+    def validate_scopes(self, value):
+        if not value:
+            return value
+
+        validation_errors = []
+        for scope in value:
+            # if the existing instance already has this scope, skip the check
+            if self.instance and self.instance.has_scope(scope):
+                continue
+            # add an error if the requester lacks permissions being requested
+            if not self.access.has_scope(scope):
+                validation_errors.append(
+                    "Requested permission of %s exceeds requester's permission. Please contact an administrator to make the requested change."
```

Comment:
`Please contact an administrator to make the requested change` is this supposed to me requested change of the role permissions? or do you mean like the actually changing of the scope?

Might be enough to say `Insufficient permissions: Your role does not have access to %s permission`

Reply:
@MeredithAnya I want the bit about contacting an administrator to let the user know that using that scope is possible but they need an admin to do it for them. My concern about saying `Insufficient permissions: Your role does not have access to %s permission` is that they might not realize there is a way to get past this if they really need to. What do you think?

Reply:
yeah I guess we can keep it. seems a little wordy to me but can't really think of anything better so...